openblas/src/main/java/org/bytedeco/openblas/presets/openblas_full.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (C) 2016-2025 Samuel Audet
+ *
+ * Licensed either under the Apache License, Version 2.0, or (at your option)
+ * under the terms of the GNU General Public License as published by
+ * the Free Software Foundation (subject to the "Classpath" exception),
+ * either version 2, or any later version (collectively, the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *     http://www.gnu.org/licenses/
+ *     http://www.gnu.org/software/classpath/license.html
+ *
+ * or as provided in the LICENSE.txt file that accompanied this code.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.bytedeco.openblas.presets;
+
+import java.util.List;
+import java.util.ListIterator;
+import org.bytedeco.javacpp.ClassProperties;
+import org.bytedeco.javacpp.annotation.NoException;
+import org.bytedeco.javacpp.annotation.Platform;
+import org.bytedeco.javacpp.annotation.Properties;
+import org.bytedeco.javacpp.presets.javacpp;
+import org.bytedeco.javacpp.tools.Info;
+import org.bytedeco.javacpp.tools.InfoMap;
+
+/**
+ *
+ * @author Samuel Audet
+ * @author Dragan Djuric
+ */
+@Properties(inherit = openblas_nolapack.class, global = "org.bytedeco.openblas.global.openblas_full", value = {
+    @Platform(
+        include = {"openblas_config.h", "cblas.h", "lapacke_config.h", "lapacke_mangling.h", "lapack.h", "lapacke.h", "lapacke_utils.h"})})
```

Comment:
Try to inherit from openblas.class instead so that we get the init() method and the include list too

Reply:
Will do. This was ommision. Just to check: It is enough to only replace the reference at this exact place?

Reply:
Should be, yes

Reply:
Thanks. Fixed.